applications/crossbar/src/modules/cb_notifications.erl

```diff
@@ -34,7 +34,9 @@
                                  ]).
 -define(CB_LIST, <<"notifications/crossbar_listing">>).
 -define(PREVIEW, <<"preview">>).
-
+-define(SMTP_LOG, <<"log">>).
```

Comment:
I think the URI should be `notifications/logs` since it is a collection of logs instead of just one log.